src/datumaro/components/project.py

```diff
@@ -382,6 +383,7 @@ def get_slice(self, target) -> Pipeline:
         return pipeline
 
 
+@deprecated(deprecated_version="1.11", removed_version="1.12")
```

Comment:
Strange that it shows this as a change, this line is already in the develop branch.